etc/profile-a-l/firefox.profile

```diff
@@ -33,6 +33,7 @@ whitelist ${HOME}/.mozilla
 # Note: Start KeePassXC before Firefox and keep it open to allow communication between them.
 #whitelist ${RUNUSER}/kpxc_server
 #whitelist ${RUNUSER}/org.keepassxc.KeePassXC.BrowserServer
+#noblacklist ${RUNUSER}/app
```

Comment:
This should also be added to {cachy-browser,floorp,librewolf}.profile.

Reply:
> This should also be added to {cachy-browser,floorp,librewolf}.profile.

If this applies to multiple browsers, I think it would be better to improve the
comment then move it to firefox-common.profile.

Then copy it to chromium-common.profile as suggested above.

I could do the refactor after this PR.